.gitlab-ci.yml

```
@@ -133,3 +133,43 @@ Restore container:
     BACKUP_DB_FILE: "hasura_prod_db.psql.gz"
     # kosko options
     KOSKO_GENERATE_ARGS: --env prod restore
+
+Generate sitemap (prod):
+  stage: Deploy
+  allow_failure: true
+  extends:
+    - .base_deploy_kosko_stage
+  rules:
+    - when: manual
+    - if: '($CI_PIPELINE_SOURCE == "trigger" || $CI_PIPELINE_SOURCE == "web") && $ACTION == "SITEMAP"'
+      when: always
+    - if: $CI_PIPELINE_SOURCE == "push"
+      when: never
+  environment:
+    name: prod2
+  variables:
+    SITEMAP_ENDPOINT: "https://cdtn-admin.fabrique.social.gouv.fr/api/sitemap"
+    DESTINATION_CONTAINER: sitemap
+    DESTINATION_NAME: sitemap.xml
+    SECRET_NAME: azure-cdtnadminprod-volume
+    KOSKO_GENERATE_ARGS: --env prod sitemap-uploader
+
+Generate sitemap (dev):
+  stage: Deploy
+  allow_failure: true
+  extends:
+    - .base_deploy_kosko_stage
+  rules:
+    - when: manual
+    - if: '($CI_PIPELINE_SOURCE == "trigger" || $CI_PIPELINE_SOURCE == "web") && $ACTION == "SITEMAP"'
+      when: always
+    - if: $CI_PIPELINE_SOURCE == "push"
+      when: never
```

```suggestion
      when: never
    - if: '($CI_COMMIT_TAG != "" || $CI_COMMIT_REF_SLUG == "master")'
      when: never
```

dont show on tags or master ?